### DISCUSSION TITLE
Error installing 4.0.2 on Debian 11: That failed! Perhaps your configuration is not right

### DISCUSSION THREAD

**Discussion by watrworld** [mastodon/mastodon#22171] [General]
Trying to install Mastodon on new server Debian 11, and I get the following to errors after

sudo -u mastodon RAILS_ENV=production bundle exec rake mastodon:setup --trace   

1) Tasks: TOP => db:setup => db:schema:load => db:check_protected_environments
(See full trace by running task with --trace)
That failed! Perhaps your configuration is not right

2) @mastodon/mastodon@: The engine "node" is incompatible with this module. Expected version ">=14". Got "12.22.12"

On (1) I did have --trace, where do I look for details? For (2), I am guessing my jsnode needs update

PS: I am roughly following https://www.youtube.com/watch?v=2a9YrLsE45Y   except w 4.0.2

Thanks for any help!

**Comment by watrworld** [comment]
Never mind, founds out 4.0.2 is not compatible with Debian 11

**Comment by wiegelmann** [comment]
Nope, I'm running Mastodon on Debian since 2018 up to the latest version, everything fine.
1. Check your config file, something is missing, maybe a typo.
2. Update your node version.
